src/Subscription.ts

```diff
@@ -126,4 +126,24 @@ export class UnsubscriptionError extends Error {
     super('unsubscriptoin error(s)');
     this.name = 'UnsubscriptionError';
   }
+}
+
+function callPrivateUnsubscribe(sub: Subscription): [boolean, any] {
```

Comment:
These external functions are going to introduce closure when they're used. If they were on the Subscription prototype, they would not.